<launch>

  <!--Launch Realsense-->
  <include file="$(find kb_utils)/launch/realsense.launch"/>

  <!--InertialSense-->
  <node name="inertial_sense" type="inertial_sense_node" pkg="inertial_sense">
  </node>

  <!--KB Driver-->
  <node name="teensy" pkg="kb_utils" type="kb_driver.py" args="/dev/ttyACM0"/>
 
  <node name="record" pkg="robag" type="record" args="-O $(arg name) 
    /encoder /sonar /ins /command /camera/color/image_raw/compressed /camera/depth/image_rect_raw/compressed" />

</launch>
